src/components/icons/EyeCrossed.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ComponentWithAs } from '@chakra-ui/system';
+import { createIcon, IconProps } from '@chakra-ui/react';
+
+export const EyeCrossed: ComponentWithAs<'svg', IconProps> = createIcon({
```

Comment:
We add `Icon` at the end of the name. Also for the file.

```suggestion
  displayName: 'EyeCrossedIcon',
```